Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -259,7 +259,10 @@ enum TrackingEvent: String, Trackable {
     // MARK: add delegate key
     case addDelegateKeyLedger                       = "screen_delegate_ledger_nano_x"
     case addDelegateKeyWalletConnect                = "screen_delegate_walletconnect"
-
+    case addDelegateKeyImported                     = "screen_delegate_imported"
+    case addDelegateKeyGenerated                    = "screen_delegate_generated"
+    case addDelegateKeyKeystone                     = "screen_delegate_keystone"
```

Comment:
@MouazAlzahabi this didn't come in the requirements, however, can we join these 3 tracking events into one, and track it under `screen_add_delegate`, and have a parameter `key_type` with one of 3 values `imported`, `generated`, `keystone`, and `walletconnect`, and `ledger`?

Reply:
(in another refactoring)

Reply:
actually, never mind, maybe we make a separate refactoring of tracking.

Reply:
I made it like this because we already had two for WalletConnect&ledger

Reply:
@DmitryBespalov done